### PR TITLE
add --no-adopt-patches

### PR DESCRIPTION
**Pull request by johnoliver**

### CODE REVIEW

**Comment by johnoliver** [comment]
The test failure should be ignored, as the PR builder has not been merged yet, that test is not valid

File: sbin/build.sh [hunk 1]
```diff
@@ -166,7 +166,7 @@ configuringVersionStringParameter()
       addConfigureArg "--with-user-release-suffix=" "${dateSuffix}"
     fi
 
-    if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ]; then
+    if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_HOTSPOT}" ] && [ ${BUILD_CONFIG[ADOPT_PATCHES]} == true ]; then
       addConfigureArg "--with-company-name=" "AdoptOpenJDK"
```
Comment (karianna) [inline sbin/build.sh]: This is going to be true for any `dev` branch build effectively yes?

Comment (karianna) [reply sbin/build.sh]: Will need to think about this with regards to say if we build Coretto.

Comment (johnoliver) [reply sbin/build.sh]: for any NON-dev branch based builds. Yes for Coretto 8 we will probably need to pass the no patch arg

Comment (karianna) [reply sbin/build.sh]: OK we can resolve this once we sort out variant.

**Pull request merged**